### DISCUSSION TITLE
Open source project and contributions but no free co-pilot offer

### DISCUSSION THREAD

**Discussion by gerstrong** [community/community#38136] [Copilot Conversations] [answered]
I would like to test co pilot and I am a bit disappointed, that I did not get any offer. As Main Developer of Commander Genius I think I could have gotten an offer. Also I have been contributing to MoneyManagerEx and the wxWidgets library lately, which also are fine OSS projects.

I love open source software and will continue contributing, but at least I would like to know why I am not considered one by Github. 

Looking forward to find out why...

**Comment by david-engelmann** [comment]
The email address of GitHub customer service is [support@github.com](mailto:support@github.com). Just give them time to respond. You can also reach out on [Twitter](https://twitter.com/github).

**Comment by iamcjwilliams** [comment]
@gerstrong Our OSS Maintainer list is updated weekly, for account-specific questions like this one open a ticket on our [Support page](https://support.github.com/contact)

**Comment by gerstrong** [comment, marked as answer]
For those who are looking for an alternative meanwhile(:-D): https://marketplace.visualstudio.com/items?itemName=BadBoy17G.clara-copilot

I contacted the support and they claim I am not an OSS Maintainer, a statement I clearly reject since Commander Genius (https://clonekeenplus.sourceforge.io/) is my main open source project I have been developing and maintaining since 2008. They did not even consider the other projects I have been contributing.

Their reason was that it has too many references to gitlab. I understand that github is not the biggest fan of competition, but I clearly believe that the OSS maintainer support claim should be taken more seriously. I saw for them a chance to really win me back, but these arguments considered, I will keep my project as is and reject there judgement me not being an OSS maintainer.

Keep up the amazing work!
